cmd/goal/account.go

```diff
@@ -31,7 +31,7 @@ import (
 	"github.com/algorand/go-algorand/config"
 	"github.com/algorand/go-algorand/crypto"
 	"github.com/algorand/go-algorand/crypto/passphrase"
-	v1 "github.com/algorand/go-algorand/daemon/algod/api/spec/v1"
+	generatedV2 "github.com/algorand/go-algorand/daemon/algod/api/server/v2/generated"
```

Comment:
Can you summarize what we're doing with versioning that forces changes in this PR?  We don't have to increment the version for changes like this, right?  Maybe you're just migrating to the new version that we previously had to do?

Reply:
Actually I had to upgrade to using the v2 endpoint because v1 does not return extra pages values for accounts.